examples/bri-3/README.md

```diff
@@ -43,6 +43,23 @@ $ npx prisma db seed # seed db
 
 Relevant information can be found in ./docs/nats/nats-configuration.md
 
+## Environment configuration
+
+Can be found in ./env.sample. Explanation: 
+
+```bash
+DATABASE_URL="postgresql://postgres:example@localhost:5432/postgres" # DB connection string
+GOERLI_RPC_URL="" # Any GOERLI RPC url i.e. "https://rpc.ankr.com/eth_goerli". This is needed in order for did authentitaction to work.
```

Comment:
more precise comment would be to resolve dids

maybe we can open issue to document did auth and do it there? i need to refresh that part a bit

Reply:
Ok, i will do that.